index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <p>Hello World, we are software developers!</p>
```

Comment:
Nice job pointing this out, the use of appropriate tags allows for better page structuring and accessibility :+1: . However, I suggest specifying the reason you requested this change.